### PR TITLE
Add topic about Reorder from the Storefront

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) adds topic about Reorder from the Storefront

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/customers/account-dashboard-my-orders.html

whatsnew
Added the [Reorders](https://docs.magento.com/user-guide//customers/account-dashboard-reorders.html) to describe the functionality for customers to reorder products on the storefront.

### CODE REVIEW

File: src/customers/account-dashboard-reorders.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Reorders
+---
+
+For the reorder to be available, make sure the [_Allow Reorder_]({% link sales/reorders-allow.md %}) configuration option is enabled.
```
Comment (leslietilling) [inline src/customers/account-dashboard-reorders.md:5]:
```suggestion
Reorders are available to customers when the [_Allow Reorder_]({% link sales/reorders-allow.md %}) configuration option is enabled.
```

File: src/customers/account-dashboard-reorders.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Reorder from My Orders page
+
+The _Reorder_ button is displayed in the list with Orders near the _View_ button.
+
+![]({% link images/images/account-dashboard-reorder.png %}){: .zoom}
+_Reorder button on the My Order page_
+
+**Case 1.** All products from the order are available for reorder
+
+User will be redirected to the [shopping cart]({% link quick-tour/shopping-cart.md %}), and all products are added to the cart
```
Comment (leslietilling) [inline src/customers/account-dashboard-reorders.md:21]:
```suggestion
Customer is redirected to the [shopping cart]({% link quick-tour/shopping-cart.md %}), and all products are added to the cart
```

File: src/customers/account-dashboard-reorders.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+The Reorder button will not appear on the _My Orders_ and _View Order_ pages.
+
+![]({% link images/images/account-dashboard-reorder-grid.png %}){: .zoom}
+_My Order page_
+
+## Reorder from Order view page
+
+**Case 1.** All products from the order are available for reorder
+
+User will be redirected to the cart, and all products are added to the cart
```
Comment (leslietilling) [inline src/customers/account-dashboard-reorders.md:34]:
```suggestion
Customer is redirected to the cart, and all products are added to the cart
```

File: src/customers/account-dashboard-reorders.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+**Case 2.** Some/all products from the order are not available for reorder
+
+The Reorder button will not appear on the My Orders and View Order pages.
```
Comment (leslietilling) [inline src/customers/account-dashboard-reorders.md:38]:
```suggestion
The _Reorder_ button does not appear on the My Orders and View Order pages.
```

File: src/customers/account-dashboard-reorders.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+{:.bs-callout-tip}
+If the cart is not empty and the user clicks Reorder (from the My Orders or Order View page), the existing products will remain in the cart with the added reorder products.
```
Comment (leslietilling) [inline src/customers/account-dashboard-reorders.md:41]:
```suggestion
If the cart is not empty and the customer clicks **Reorder** (from the My Orders or Order View page), the existing products remain in the cart with the added reorder products.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edits

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you for reviewing ☀️
I have implemented your suggestions, could you please approve this PR?
Thank you!

File: src/customers/account-dashboard-reorders.md (hunk 5 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-reorders.md]: Done ✅

File: src/customers/account-dashboard-reorders.md (hunk 4 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-reorders.md]: Done ✅

File: src/customers/account-dashboard-reorders.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-reorders.md]: Done ✅

File: src/customers/account-dashboard-reorders.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-reorders.md]: Done ✅

File: src/customers/account-dashboard-reorders.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-reorders.md]: Done ✅

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**